src/Tribe/Status/Abstract_Commerce.php

```diff
@@ -227,16 +229,10 @@ public function get_available_incomplete_counts_for_ticket( $ticket_sold ) {
 	 * @return string a string of html for the tooltip
 	 */
 	public function get_sale_by_ticket_tooltip() {
-		ob_start();
-		?>
-		<div class="tribe-tooltip" aria-expanded="false">
-			<span class="dashicons dashicons-info"></span>
-			<div class="down">
-				<?php echo esc_html__( 'Sold counts tickets from completed orders only.', 'event-tickets-plus' ); ?><i></i>
-			</div>
-		</div>
-		<?php
-		return ob_get_clean();
+
+		$message = esc_html__( 'Sold counts tickets from completed orders only.', 'event-tickets' );
+
+		return tribe( 'tooltip.view' )->render_tooltip( $message, [] );
```

Comment:
Totally not wrong but the second param is optional 😉

Reply:
good call, removed them if empty